pkg/controllers/execution/execution_controller.go

```diff
@@ -253,8 +261,31 @@ func (c *Controller) tryCreateOrUpdateWorkload(ctx context.Context, clusterName
 	return nil
 }
 
+func (c *Controller) updateWorkDispatchingConditionIfNeeded(ctx context.Context, work *workv1alpha1.Work) error {
+	newWorkDispatchingCondition := metav1.Condition{
+		Type:               workv1alpha1.WorkDispatching,
+		LastTransitionTime: metav1.Now(),
+	}
+
+	if helper.IsWorkSuspendDispatching(work) {
+		newWorkDispatchingCondition.Status = metav1.ConditionFalse
+		newWorkDispatchingCondition.Reason = workSuspendDispatchingConditionReason
+		newWorkDispatchingCondition.Message = "Work dispatching is in a suspended state."
```

Comment:
Should we record the names of the member clusters here?

Reply:
The cluster information may be obtained from the namespace of work.

Reply:
> The cluster information may be obtained from the namespace of work.

I know what you mean - if the cluster name is also present in the condition, I think it would be more intuitive.

Reply:
You have a point.
Hi @a7i, would you be willing to change it, or shall we do it separately in the next pr.

Reply:
Let's see if we need the cluster info in the message.

This kind of engineering fly trap, `this could be useful`. 
 In my opinion, the cluster name might be redundant. I think we can postpone it until there is more clear evidence.